config/consensus.go

```diff
@@ -562,6 +562,12 @@ type ConsensusParams struct {
 
 	// EnableSha512BlockHash adds an additional SHA-512 hash to the block header.
 	EnableSha512BlockHash bool
+
+	// EnableInnerClawbackWithoutSenderHolding allows an inner clawback (axfer
+	// w/ AssetSender) even if the Sender holding of the asset is not
+	// available. This parameters can be removed and assumed true after the
+	// first consensus release in which it is set true.
+	EnableInnerClawbackWithoutSenderHolding bool
```

Comment:
Just want to make sure I am reading this correctly. 

```
sender: Alice,
assetSender: Bob
```

In this case we would need the holding for Bob but not Alice. What's the behavior now, needing both Alice and Bob?

Reply:
Right, it currently needs both, which probably hasn't tripped anyone up because you usually have the sender and ASA available in the caller's foreign arrays  which makes the holding available. But with tx.Access, listing resources is more fine-grained. We don't want cakes to need more in there than necessary.